core-api/src/main/java/com/optimizely/ab/internal/ExperimentUtils.java

```diff
@@ -101,9 +105,11 @@ public static boolean isUserInExperiment(@Nonnull ProjectConfig projectConfig,
 
         Condition conditions = experiment.getAudienceConditions();
         if (conditions == null) return null;
-
+        logger.debug(String.format("Evaluating audiences for experiment \"%s\": \"%s\"", experiment.getKey(), conditions.toString()));
         try {
-            return conditions.evaluate(projectConfig, attributes);
+            Boolean result = conditions.evaluate(projectConfig, attributes);
+            logger.info(String.format("Audiences for experiment %s collectively evaluated as " + result, experiment.getKey()));
```

Comment:
not justified as `info` level IMO.

I would rather leave it up to the user of SDK to log this on the caller side than to put here.